image/src/teachable-mobilenet.ts

```diff
@@ -309,18 +305,17 @@ export class TeachableMobileNet extends CustomMobileNet {
         })
         */
 
-        const history = await trainingModel.fitDataset(trainData, {
+        const history = await this.trainingModel.fitDataset(trainData, {
             epochs: params.epochs,
             validationData,
             callbacks
         });
 
         const jointModel = tf.sequential();
```

Comment:
should we be calling `jointModel.dipose()`? we call it on truncated and training but not on the joint? If we call it on the joint wouldnt that be also calling it on truncated and training?

Reply:
Just making a note here based on what we chatted. We can't call dispose on `jointModel` because it hasn't been compiled. Will throw a missing optimizer error. But we can call dispose on `TeachableMobileNet` through the extended class.